docs/UserGuide.md

```diff
@@ -117,7 +117,30 @@ The following points explain the format of a command.
   e.g. if there exists a company named `meta`, adding another company with the name `meta` is not allowed. 
   
 --------------------------------------------------------------------------------------------------------------------
+# Demo Use Cases <a id="demos"></a>
 
+## Use Case #1: Applying to a role in a company
+
+Say you found a company with an opening that you are interested in. You did your research and found relevant information pertaining to the company and the role. Here's how you can input this data in the application:
+
+1. [Add the company](#c-add-c) to the company list.
```

Comment:
I thinking if it will be clearer if we highlight the display box that this is the output you should see if it is successful. I scare the reader will be confused why the display box already has output before you enter the command? If you think is clear then can just leave it.

Reply:
Gotcha 👍